AirCasting/CreateSessionViews/ConnectingABView/AirBeamConnectionController.swift

```diff
@@ -0,0 +1,33 @@
+// Created by Lunar on 30/07/2021.
+//
+
+import Foundation
+import CoreBluetooth
+
+enum AirBeamServicesConnectionResult {
```

Comment:
let's move this to the same file as `ConnectionAirBeamServices` since this detail is not important for `AirBeamConnectionController` users at all